_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -629,7 +629,11 @@ mfxStatus ImplementationAvc::QueryIOSurf(
         checkSts = lpSts;
 
     SetDefaults(tmp, hwCaps, true, core->GetHWType(), core->GetVAType(), *QueryCoreInterface<eMFXGTConfig>(core, MFXICORE_GT_CONFIG_GUID));
+
+#ifdef MFX_VA_WIN
     mfxExtCodingOption3 const &   extOpt3 = GetExtBufferRef(tmp);
```

Comment:
extOpt3 is not used: there is no path for MFX_VA_WIN which uses it. Just remove.